docs/Wetboek Tedeapolis.md

```diff
@@ -0,0 +1,566 @@
+# Wetboek Tedeapolis 2020
+
+## Wetstechnische informatie
+
+| | |
+|---|---|
+| Overheidsorganisatie      | Hoge Raad Tedeapolis |
+| Officiële naam            | Algemeen Wetboek Tedeapolis 2020 |
+| Citeertitel               | Wetboek Tedeapolis|
+| Vastgesteld door          | De Hoge Raad|
+| Onderwerp                 | Openbare orde en veiligheid|
+
+## Algemene Bepalingen
+
+### A1 - Reikwijdte
+
+1. Het Wetboek Tedeapolis (verder aangeduid als WTDA) is bedoeld als aanvulling op het Nederlands Recht en heeft betrekking op het eiland, de gemeente, en de stad Tedeapolis.
+2. Het Wetboek is leidend voor alle beschreven gevallen. Daarbuiten is het Nederlands Recht van toepassing.
+3. Het Wetboek is nauw verweven met de Algemene Plaatselijke Verordening Tedeapolis (APV). De APV biedt echter slechts een indicerend kader voor eventuele bestraffing in-game (“in RP”) en is verder bedoeld voor het afhandelen van straffen buiten de RP om.
+
+### A2 - Geldigheid
+
+1. Bij veranderingen van een wetgeving na het tijdstip waarop het feit begaan is, worden de voor de verdachte meest gunstigste bepalingen toegepast.
+2. Het is voor een staff-lid niet toegestaan om mee te beslissen over een zaak die over hunzelf, vrienden en/of familie gaat. Dit om te versterken dat een zaak op een eerlijke en rechtvaardige manier behandeld wordt.
+3. Een overtreding of misdrijf die niet opgenomen is in het Wetboek Tedeapolis, maar wel als zodanig bekend staat in het Nederland Recht, kan niet bestraft worden.
+   * Wel kan een dergelijke overtreding of misdrijf leiden tot een notitie, ten behoeve van de politieadministratie.
+
+### A3 - Strafbepaling
+
+1. Straffen zijn vastgesteld en opgenomen per artikel. Hieronder vallen:
+   * Waarschuwingen;
+   * Taakstraffen;
+   * Celstraffen;
+   * rijontzeggingen;
+   * inbeslagnames of invorderingen;
+   * Boetes.
+2. Een opgelegde straf kan een combinatie zijn van de bovenstaande strafvormen.
+3. Een opgelegde straf is nooit hoger dan de vastgestelde waarden.
+4. Een straf kan wel verhoogd worden door het stapelen van overtredingen.
+5. Een straf kan ook verlaagd worden door verzachtende omstandigheden.
+6. Een straf kan geheel of gedeeltelijk kwijtgescholden of uitgesloten worden op basis van uitsluitingsgronden of bewijsbaar foutieve uitschrijving van de straf.
+
+### A4 - Strafblad
+
+1. Alleen straffen die enige vorm van celstraf bevatten kunnen leiden tot een strafblad.
+2. Een strafblad blijft staan voor een periode van 2 (realtime) maanden, ingaand op de datum van de opgelegde celstraf.
```

Comment:
Dit klopt voor hoever ik weet niet helemaal, een strafblad is 1 maand geldig tenzij dit om een geweldsdelict ging. Dan is de duur 3 maanden.

Reply:
Dank! Dan wijzigen we dat.